homeassistant/components/lamarzocco/button.py

```diff
@@ -56,3 +56,4 @@ class LaMarzoccoButtonEntity(LaMarzoccoEntity, ButtonEntity):
     async def async_press(self) -> None:
         """Press button."""
         await self.entity_description.press_fn(self.coordinator.device)
+        self.async_write_ha_state()
```

Comment:
this is since the method is now changing an attribute in the device `config`

Reply:
Euh I dont see why this matters?

Reply:
Because somebody might have asked why this was added

Reply:
Yes, that was my question haha

Reply:
> since the method is now changing an attribute in the device `config`

😅 
The button is directly changing the state of the new binary sensor now, so I'm writing the state at press now

Reply:
Yes, busy async write ha state is only for the current state right?